src/internal/pfsdb/branches.go

```diff
@@ -133,9 +133,9 @@ func NewBranchIterator(ctx context.Context, tx *pachsql.Tx, startPage, pageSize
 	}
 	query := getBranchBaseQuery
 	if len(conditions) > 0 {
-		query += fmt.Sprintf("\nWHERE %s", strings.Join(conditions, " AND "))
+		query += "\n" + fmt.Sprintf("WHERE %s", strings.Join(conditions, " AND "))
```

Comment:
Does this change the semantics of the code at all?

Reply:
No, but I wanted to make it more explicit to future programmers that it is important to separate out the sections with either space or newlines.